equinox is failing with:
```
=========================== short test summary info ============================
FAILED tests/test_errors.py::test_traceback_runtime_eqx - assert 3 == 2
======= 1 failed, 390 passed, 1 skipped, 57 warnings in 64.44s (0:01:04) =======
```

See complete logs [here](https://pastebin.com/XqcqUHvd).

Any idea @patrick-kidger

--- a/pkgs/development/python-modules/equinox/default.nix
+++ b/pkgs/development/python-modules/equinox/default.nix
@@ -72,6 +72,10 @@ buildPythonPackage rec {
     # AssertionError: assert 'foo:\n   pri...pe=float32)\n' == 'foo:\n   pri...pe=float32)\n'
     # Also reported in patrick-kidger/equinox#716
     "test_backward_nan"
+
+    # Flaky test
+    # See https://github.com/patrick-kidger/equinox/issues/781
+    "test_traceback_runtime_eqx"
   ];
 
   meta = {

--- a/pkgs/development/python-modules/jaxtyping/default.nix
+++ b/pkgs/development/python-modules/jaxtyping/default.nix
@@ -18,7 +18,7 @@
 let
   self = buildPythonPackage rec {
     pname = "jaxtyping";
-    version = "0.2.31";
+    version = "0.2.33";
     pyproject = true;
 
     disabled = pythonOlder "3.9";
@@ -27,7 +27,7 @@ let
       owner = "google";
       repo = "jaxtyping";
       rev = "refs/tags/v${version}";
-      hash = "sha256-kuGFzp8sDLq6J/qq8ap3lD3n1pABHurXcbRUtDQyWwE=";
+      hash = "sha256-CL1EONbjjT3SCAn2o1x+1cgfuYWMEgQwbX9j34t+HGs=";
     };
 
     build-system = [

